Error Cannot Find Package

:go-ps4 (master)$ go run cmd/ps4/main.go 
```
cmd/ps4/main.go:12:2: cannot find package "github.com/lucasepe/playstation" in any of:
	/snap/go/4098/src/github.com/lucasepe/playstation (from $GOROOT)
	/home/fathan/work/go/src/github.com/lucasepe/playstation (from $GOPATH)
cmd/ps4/main.go:13:2: cannot find package "github.com/lucasepe/playstation/utils" in any of:
	/snap/go/4098/src/github.com/lucasepe/playstation/utils (from $GOROOT)
	/home/fathan/work/go/src/github.com/lucasepe/playstation/utils (from $GOPATH)
```

--- a/cmd/ps4/main.go
+++ b/cmd/ps4/main.go
@@ -9,8 +9,8 @@ import (
 	"path/filepath"
 	"strings"
 
-	ps "github.com/lucasepe/playstation"
-	"github.com/lucasepe/playstation/utils"
+	ps "github.com/lucasepe/go-ps4"
+	"github.com/lucasepe/go-ps4/utils"
 )
 
 var (

